what_is.md

```diff
@@ -355,10 +363,10 @@ tree sequence and the underlying biological processes that produced the genetic
 sequences in the first place, such as those pictured in the demography above. For
 example, each branch point (or "internal node") in one of our trees can be
 imagined as a genome which existed at a specific time in the past, and
-which is a "most recent common ancestor" (MRCA) of the descendant genomes at that
-position on the chromosome. We can mark these extra "ancestral genomes" on our tree
-diagrams, distinguishing them from the *sampled* genomes ($\mathrm{a}$ to $\mathrm{j}$)
-by using circular symbols. We can even colour the nodes by the population that we know
+which is a MRCA of the descendant genomes at that position on the chromosome.
```

Comment:
We don't define MRCA anywhere previously, do we? So I think we do need to spell it out here.